wormhole-connect/src/routes/operator.ts

```diff
@@ -284,19 +259,44 @@ export class Operator {
   ): Promise<TokenConfig[]> {
     const supported: { [key: string]: TokenConfig } = {};
     for (const route of config.routes) {
-      for (const token of config.tokensArr) {
-        const { key } = token;
-        const alreadySupported = supported[key];
-        if (!alreadySupported) {
-          const isSupported = await this.isSupportedDestToken(
-            route as Route,
-            config.tokens[key],
-            sourceToken,
-            sourceChain,
-            destChain,
-          );
-          if (isSupported) {
-            supported[key] = config.tokens[key];
+      if (!config.routes.includes(route)) {
```

Comment:
I think we can remove the `if` here

Reply:
The `for...of` loop already iterates over each `route` in `config.routes` unless I'm missing something

Reply:
ah right. thanks.